cdk/variance/config.yaml

```diff
@@ -0,0 +1,19 @@
+message:
+  # Here since supportedMessageTypes is not present
+  # We'd assume that the below mentioned transformation
+  # is valid for all message types
+  default:
+    transformation:
+      # Here first the message would be spread on that
+      # the mappings would be overridden
+      # ResponsePayload = { ...message, ...mappedPayload }
+      spreadMessage: true
+      mapperPath: ./mappings/default.yaml
```

Comment:
What is the need to explicitly mention it if it can be auto-deduced?
CC @sanpj2292

Reply:
With this path thing, we're giving user the flexibility as that they can mention the yaml anywhere inside the <destination> folder
I mean inside the folder `/v0/destinations/<destination>` they can mention the mappings yaml anywhere they like & with whatever name they're comfortable with

Reply:
It makes sense to specify when it is different from anything other than the defaults (default.yaml, track.yaml, identify.yaml, etc under mappings directory).

Reply:
That decision is left to the user or developer

Reply:
I didn't see any piece of code in the CDK for assuming the defaults (till now). Anyway, in this case, it is redundantly specified in the config.

Reply:
I don't understand what defaults are you pointing about ?